open_learning_ai_tutor/Tutor.py

```diff
@@ -98,16 +98,16 @@ class GraphTutor2(Tutor):
     def __init__(
         self,
         client,
-        pb,
-        sol,
+        problem,
+        problem_set,
         model="gpt-4o-mini",
         intermediary=None,
         intent_history=[],
         assessment_history=[],
         tools=None,
         options=dict(),
     ) -> None:
-        self.pb, self.sol = pb, sol
+        self.problem, self.problem_set = problem, problem_set
```

Comment:
GraphTutor2 subclasses Tutor - consider making the method signatures the same and consolidating the common logic in the super class

Reply:
This code is part of the code that will be deleted

Reply:
open_learning_ai_tutor/Tutor.py will have a pretty significant rewrite once i get to it